pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -1,124 +1,105 @@
+PauliZ                 = { properties = [ "invertible",                 "differentiable" ] }
+PhaseShift             = { properties = [ "invertible",                 "differentiable" ] }
+QubitUnitary           = { properties = [ "invertible",                                  ] }
+Rot                    = { properties = [ "invertible",                                  ] }
+RX                     = { properties = [ "invertible",                 "differentiable" ] }
+RY                     = { properties = [ "invertible",                 "differentiable" ] }
+RZ                     = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitationMinus  = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitationPlus   = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitation       = { properties = [ "invertible",                 "differentiable" ] }
+S                      = { properties = [ "invertible",                 "differentiable" ] }
+SWAP                   = { properties = [ "invertible",                 "differentiable" ] }
+Toffoli                = { properties = [ "invertible",                 "differentiable" ] }
+T                      = { properties = [ "invertible",                 "differentiable" ] }
 
 # Operators that should be decomposed according to the algorithm used
 # by PennyLane's device API.
 # Optional, since gates not listed in this list will typically be decomposed by
 # default, but can be useful to express a deviation from this device's regular
 # strategy in PennyLane.
-decomp = [
-        "BasisState",
-        "QubitStateVector",
-        "StatePrep",
-        "QFT",
-        "MultiControlledX",
-]
+[operators.gates.decomp]
+
+BasisState             = {}
+MultiControlledX       = {}
+QFT                    = {}
+QubitStateVector       = {}
+StatePrep              = {}
 
 # Gates which should be translated to QubitUnitary
-matrix = [
-        "ControlledQubitUnitary",
-        "ECR",
-        "SX",
-        "ISWAP",
-        "PSWAP",
-        "SISWAP",
-        "SQISW",
-        "CPhase",
-        "OrbitalRotation",
-        "QubitCarry",
-        "QubitSum",
-        "DiagonalQubitUnitary",
-        "BlockEncode",
-]
+[operators.gates.matrix]
+
+BlockEncode            = {}
+CPhase                 = {}
+DiagonalQubitUnitary   = {}
+ECR                    = {}
+ISWAP                  = {}
+OrbitalRotation        = {}
+PSWAP                  = {}
+QubitCarry             = {}
+QubitSum               = {}
+SISWAP                 = {}
+SQISW                  = {}
+SX                     = {}
+
+# Observables supported by the device
+[operators.observables]
+
+Identity               = {}
+PauliX                 = {}
+PauliY                 = {}
+PauliZ                 = {}
+Hadamard               = {}
+Hermitian              = {}
+Hamiltonian            = {}
+SparseHamiltonian      = {}
```

Comment:
No tensor observable support on the kokkos device?

Reply:
There is, it is part of the [base](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/core/src/observables/Observables.hpp#L318) API.

Reply:
FYI kokkos device [contains](https://github.com/PennyLaneAI/pennylane-lightning/blob/50864f505071ffb8a67430cebc93a019eee0a076/pennylane_lightning/core/src/simulators/lightning_kokkos/observables/tests/Test_ObservablesKokkos.cpp#L94) tests mentioning tensor observables

Reply:
Right so let's add it :)

Reply:
Added [here](https://github.com/PennyLaneAI/pennylane-lightning/pull/642/files#diff-45f8d82a726b88e990143dd273747c086a5dcfaec6a302400d07e0c25e6d1415R88)